mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -5221,6 +5221,30 @@ pub enum CancelBy {
     Coin { ticker: String },
 }
 
+pub async fn get_matching_orders(ctx: &MmArc, ticker: &str) -> Vec<Uuid> {
+    let mut matching_orders = vec![];
+    if let Ok(ordermatch_ctx) = OrdermatchContext::from_ctx(ctx) {
+        let maker_orders = ordermatch_ctx.maker_orders_ctx.lock().orders.clone();
+        let taker_orders = ordermatch_ctx.my_taker_orders.lock().await;
+
+        for (uuid, order) in maker_orders.iter() {
+            let uuid = *uuid;
```

Comment:
you can dereference while pushing to the vec `matching_orders.push(*uuid);`, please remove this.